### ISSUE TITLE
Automatically enable code block highlighting for code blocks with specified language

### ISSUE THREAD

**Issue by naught101** [vim-pandoc/vim-pandoc-syntax#150]
It'd be nice if codeblocks like:

<code>```python
blah blha
```</code>

Were automatically highlighted using the python syntax (e.g. `:PandocHighlight python`). It'd be great if it was possible to do this with docs with multiple languages, but considering how rare that is, it's probably not really necessary. Maybe just enable code syntax when the first code block is parsed?

**Comment by naught101** [comment]
Also, 

<pre>
```{python}
blah
```
</pre>

is a legitimate indicator that a codeblock is a python codeblock, but for me, it's not matching. Using vim-pandoc-syntax from git, just pulled.

**Comment by Melkster** [comment]
Yeah, [vim-markdown](https://github.com/plasticboy/vim-markdown) does this automatically. I'm considering moving from vim-markdown to vim-pandoc-syntax but this is a feature I would really miss if I switched.

**Comment by alerque** [comment]
I would love to see this enabled in this plugin too. Right now I switch back and forth between the two syntax options depending on the project because they have different strengths and weaknesses. Adding language dependent code block syntax highlighting to this plugin would go a long way towards making that unecessary.

If anybody cares to contribute this know that I'll definitely be reviewing PRs with a favorable eye toward merging!
